main.py

```diff
@@ -87,15 +100,13 @@ def main():
     r=check_releasability(repo, version, headers)
     if releasability_passed(r):
         r=do_release(repo, build_number, headers)
-        if r.status_code == 200:
-            set_release_output(f"{repo}:{version} RELEASED")
-        else:
-            print(f"::error Unexpected exception occurred while calling release cloud function. Status '{r.status_code}': '{r.text}'")
-            abort_release(repo, version)
+        validate_gcf_call(r, repo, version, "release ...")
+        if repo.endswith("sonar-enterprise"):
+            r=do_distribute(repo, build_number, headers)
```

Comment:
This should not be specific to sonarqube but optional to any project, and set at workflow definition level (in the release.yml with a distribute=true or something similar)

Reply:
I add a input to the action